docs/email/MX-Backup.md

```diff
@@ -0,0 +1,161 @@
+---
+author:
+    name: Linode Community
+    email: docs@linode.com
+description: 'Quick introduction into hosting a Email Backup server with imapsync.'
+keywords: 'email, backup, postfix, dovecot, mysql, imapsync, mx, email backup'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Weekday, September 21st, 2015'
+modified: Weekday, September 21st, 2015
+modified_by:
+    name: Linode
+title: 'Creating a E-Mail Backup Server'
+contributor:
+    name: Constantin Jacob
+    link: https://www.twitter.com/Tzeejay
+external-resources:
+	- '[IMAPSYNC](http://imapsync.lamiral.info/)'
+	- '[IMAPSYNC Github](https://github.com/imapsync/imapsync)'
+---
+
+{: .note}
+>
+> This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you're not familiar with the `sudo` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+
+Before beginning this guide you should ask yourself two questions:
```

Comment:
The entire introduction needs to be rewritten. You should begin by explaining what the software you're installing is and why you would want to install it. Avoid asking questions and then answering them, just state the facts. Also there are huge formatting issues here, concerning sentence structure, line breaks, and capitalization.